modules/osdk-upgrading-v130-to-v180.adoc

```diff
@@ -0,0 +1,895 @@
++
+[source,yaml]
+----
+version: 3-alpha
+...
+plugins:
+  manifests.sdk.operatorframework.io/v2: {}
+  scorecard.sdk.operatorframework.io/v2: {}
+----
+
+.. The `PROJECT` config version `3-alpha` must be upgraded to `3`. The `version` key in your `PROJECT` file represents the `PROJECT` config version:
++
+.Old `PROJECT` file
+[source,yaml]
+----
+version: 3-alpha
+resources:
+- crdVersion: v1
+...
+----
++
+Version `3-alpha` has been stabilized as link:https://book.kubebuilder.io/migration/v2vsv3.html[version 3] and contains a set of config fields sufficient to fully describe a project. While this change is not technically breaking because the spec at that version was alpha, it was used by default in `operator-sdk` commands, so it should be marked as breaking and have a convenient upgrade path.
+
+... Run the `alpha config-3alpha-to-3` command to convert most of your `PROJECT` file from version `3-alpha` to `3` and output comments with directions where automatic conversion is not possible:
++
+[source,terminal]
+----
+$ operator-sdk alpha config-3alpha-to-3
+----
++
+.Example output
+[source,terminal]
+----
+Your PROJECT config file has been converted from version 3-alpha to 3. Please make sure all config data is correct.
+----
+
+... Verify the change:
++
+.New `PROJECT` file
+[source,yaml]
+----
+version: "3"
+resources:
+- api:
+  crdVersion: v1
+...
+----
+
+. Make the following changes to your `config/manager/manager.yaml` file:
+
+.. For Ansible- and Helm-based Operator projects, add liveness and readiness probes.
++
+New projects built with the Operator SDK have the probes configured by default. The endpoints `/healthz` and `/readyz` are available now in the provided image base. You can update your existing projects to use the probes by updating the `Dockerfile` to use the latest base image, then add the following to the `manager` container in the `config/manager/manager.yaml` file:
++
+.Example for Ansible-based Operator projects
+[%collapsible]
+====
+[source,yaml]
+----
+  livenessProbe:
+    httpGet:
+      path: /healthz
+      port: 6789
+    initialDelaySeconds: 15
+    periodSeconds: 20
+  readinessProbe:
+    httpGet:
+      path: /readyz
+      port: 6789
+    initialDelaySeconds: 5
+    periodSeconds: 10
+----
+====
++
+.Example for Helm-based Operator projects
+[%collapsible]
+====
+[source,yaml]
+----
+  livenessProbe:
+    httpGet:
+      path: /healthz
+      port: 8081
+    initialDelaySeconds: 15
+    periodSeconds: 20
+  readinessProbe:
+    httpGet:
+      path: /readyz
+      port: 8081
+    initialDelaySeconds: 5
+    periodSeconds: 10
+----
+====
+
+.. For Ansible- and Helm-based Operator projects, add security contexts to your manager's deployment.
++
+In the `config/manager/manager.yaml` file, add the following security contexts:
++
+.`config/manager/manager.yaml` file
```

Comment:
"Example" after the dot? (Or maybe it's not an example...)

Reply:
Went back and forth on whether to include "Example" uniformly to all the code blocks or not, but in most cases they were not so much examples as they were "make the file look like this" snips.